CodingTracker.Speedierone/CodingTracker.Speedierone/UserInput.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.Data.Sqlite;
```

Comment:
Please add line between using and namespace as per code convention: https://thecsharpacademy.com/article/58